pkg/resource/deploy/deploytest/pluginhost.go

```diff
@@ -179,3 +179,11 @@ func (host *pluginHost) GetRequiredPlugins(info plugin.ProgInfo,
 	kinds plugin.Flags) ([]workspace.PluginInfo, error) {
 	return nil, nil
 }
+
+func (host *pluginHost) PolicyAnalyzer(name tokens.QName, path string) (plugin.Analyzer, error) {
```

Comment:
Nit: it would be nice to have some sort of implementation of this for testing purposes.

Reply:
I think we should do this, but we should follow up when we do the "testing" part of the next milestone.

Reply:
I would prefer that we follow up on this ASAP s.t. we can add tests to `engine/lifecycle_test` and make sure that we don't break this unintentionally.

Reply:
(so feel free to do this as a follow-up PR, but I'd rather we did so this milestone)

Reply:
lol we will for sure get to it this milestone.